tools/scoring/score/dimensions/entity_point_identification.py

```diff
@@ -26,6 +26,9 @@
 class EntityPointIdentification(Dimension):
   """Quantifies whether the proposed file
   included the correct points in each entity."""
+
+  category = DimensionCategories.COMPLEX
```

Comment:
only translations are simple?
do we need to distinguish?

Reply:
Correct. I can comment above each of these assignments.